### DISCUSSION TITLE
How get classic HTML/JavaScript/CSS files?

### DISCUSSION THREAD

**Discussion by MRoose** [graphql/graphiql#3583] [Q&A / Support]
Hello.
Don't rush to fall over laughing, I'm a backend developer on Java with some knowledge of the JavaScript and Node.js.
I have a backend service with GraphQL-server functionality and want has embedded GraphiQL UI.
I have already done this by taking the version Unpkg (CDN) from examples.
Now I need to get rid of CDN, so that all the necessary files are in the resources of my service.
My service by some endpoint return HTML-file, therefore there should be no use of Node.js, TS and etc.
Did I understand correctly, after cloning I need to go to the folder `examples/graphiql-create-react-app` and run the command `yarn build`?

**Comment by acao** [comment]
@MRoose please follow the instructions in the readme for the UMD bundle aka CDN mode!
